### DISCUSSION TITLE
第三者レビュー メッセージ-ADR

### DISCUSSION THREAD

**Discussion by creature-water-valley** [ws-4020/mobile-app-crib-notes#572] [General]
#### メッセージ管理の方針

- 決定
  - 「サーバから取得する方式を採用せず」とありますがそれは議論の対象外だったのでしょうか。コンテキストではサーバで管理する方式のメリットも書いていたので、気になりました。

**Comment by creature-water-valley** [comment]
### 対応内容

- 「コスト削減を理由として、サーバからメッセージ一覧を取得する方式は不採用」とする議論を追加

**Comment by kozake** [comment]
### 対応PR

https://github.com/ws-4020/mobile-app-crib-notes/pull/559

**Comment by creature-water-valley** [comment]
対応済みなのでクローズ
